packages/arb-ts/src/lib/bridge_helpers.ts

```diff
@@ -669,12 +666,11 @@ export class BridgeHelper {
 
     console.log('got proof')
 
-    const outboxExecuteTransactionReceipt = await BridgeHelper.tryOutboxExecute(
+    return await BridgeHelper.tryOutboxExecute(
```

Comment:
should we just return the promise instead of `return await`?
I don't think it makes a difference in this case

Reply:
agreed